variables.tf

```diff
@@ -1,10 +1,11 @@
 variable "logs_cloudwatch_retention" {
   description = "Specifies the number of days you want to retain log events in the log group."
-  default     = 90
-  type        = "string"
+  #  default     = 90
+  type = string
 }
 
 variable "zone_id" {
   description = "Route53 zone ID."
-  type        = "string"
+  type        = string
+  default     = ""
```

Comment:
Why add a default here? The default won't work because it's not a valid zone id.

Reply:
Cruft from me trying to get it to work initially. Sorry!